icmpv6.rflx

```diff
@@ -0,0 +1,105 @@
+package ICMPv6 is
+
+   type Reserved_8 is range 0 .. 0 with Size => 8;
```

Comment:
What do you think of adding the name or a link to the RFC to each package?

Reply:
We should do that. Changed in 601015bb.